### PR TITLE
DCMARWEB177 - Add Image Field

### PR DESCRIPTION
**Pull request by kNaugle03**
Includes a basic template update to output the new 404 image

### CODE REVIEW

File: digitalconvergence_4xx.module [hunk 1]
```diff
@@ -88,11 +99,35 @@ function digitalconvergence_4xx_system_site_information_settings_submit(array $f
...
   \Drupal::configFactory()
     ->getEditable('system.site')
     ->set('page.403_to_404_anonymous', $site_403_to_404_anonymous)
     ->set('page.404_title', $site_404_title)
     ->set('page.404_body', $site_404_content_body)
+    ->set('page.404_image', $site_404_image)
     ->save();
 }
 
+function digitalconvergence_4xx_preprocess_digitalconvergence_404(&$variables) {
```
Comment (amcgowanca) [inline digitalconvergence_4xx.module:118]: @kNaugle03 Where is the docblock?

File: digitalconvergence_4xx.module [hunk 2]
```diff
@@ -88,11 +99,35 @@ function digitalconvergence_4xx_system_site_information_settings_submit(array $f
...
+  $config = \Drupal::configFactory()->getEditable('system.site');
+
+  $media = \Drupal\media\Entity\Media::load($config->get('page.404_image'));
```
Comment (amcgowanca) [inline digitalconvergence_4xx.module]: @kNaugle03 What if `page.404_image` is not specified?

**Review by amcgowanca — CHANGES_REQUESTED** [review]

File: digitalconvergence_4xx.module (hunk 1 above)
Comment (kNaugle03) [reply digitalconvergence_4xx.module:118]: Added PHPDoc

File: digitalconvergence_4xx.module (hunk 2 above)
Comment (kNaugle03) [reply digitalconvergence_4xx.module]: Added a check for config exists before trying to load Media

**Pull request merged**